__tests__/app.test.js

```diff
@@ -138,6 +137,28 @@ describe("/api", () => {
             });
         });
       });
+      describe("/comments", () => {
+        describe("GET", () => {
+          test("200 - recieves back a list of comments on the article", () => {
+            return request(app)
+              .get("/api/articles/1/comments")
+              .expect(200)
+              .then(({ body: { comments } }) => {
+                comments.forEach((comment) => {
+                  expect(comment).toEqual(
+                    expect.objectContaining({
+                      body: expect.any(String),
+                      votes: expect.any(Number),
+                      author: expect.any(String),
+                      comment_id: expect.any(Number),
+                      created_at: expect.stringMatching(/(\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}.\d{3}Z)/),
+                    })
+                  );
```

Comment:
Missing tests:

- If valid and existing article doesn't have any comments
- If valid article ID but not existing article is requested